cluster/src/main/java/org/apache/iotdb/cluster/log/manage/RaftLogManager.java

```diff
@@ -677,6 +677,9 @@ public boolean matchTerm(long term, long index) {
    */
   void applyEntries(List<Log> entries) {
     for (Log entry : entries) {
+      if (entry.isApplied()) {
```

Comment:
I have added these comments: 
For add/remove logs in data groups, this log will be applied immediately when it is appended to the raft log. In this case, it will apply a log that has been applied.